SU-HSDO/suhumsci
# SHS-5543: Hover state link text for 3rd level menu items do not meet color contrast when displayed in the Mobile Menu

# READY FOR REVIEW

## Summary
Fix hover state link text for 3rd level menu items to meet color contrast on Mobile

## Need Review By (Date)
03/20

## Urgency
medium

## Steps to Test
1. Go to each of the following sites

- https://dennylab.suhumsci.loc/ - Warbler (traditional)
- https://memorylab.suhumsci.loc/ - Canyon (colorful)
- https://religiousstudies.suhumsci.loc/ - Mountain (colorful)
- https://em1060.suhumsci.loc/ - Blue jay (traditional)

2. In each of them, confirm the hover state link text for 3rd level menu items look good and meet color contrast in all resolutions

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Humsci Basic PR Checklist](https://github.com/SU-HSDO/suhumsci/blob/develop/docs/HumsciBasicPRChecklist.md)
